test/screenplay/places/builder_test.exs

```diff
@@ -0,0 +1,307 @@
+      expect(Screenplay.Stops.Mock, :fetch_parent_stops, 2, fn _stop_ids ->
+        []
+      end)
+
+      expect(Screenplay.Stops.Mock, :fetch_all_parent_stations, 1, fn ->
+        [%{"id" => "place-mvbcl", "attributes" => %{"name" => "Maverick"}}]
+      end)
+
+      expect(Screenplay.Routes.Mock, :fetch_routes_for_stop, 1, fn _ ->
+        [%{"id" => "Blue", "attributes" => %{"type" => 1}}]
+      end)
+
+      assert {:noreply, _} = Builder.handle_info(:build, state)
+
+      assert [
+               %Place{
+                 description: nil,
+                 id: "place-mvbcl",
+                 name: "Maverick",
+                 routes: ["Blue"],
+                 screens: []
+               }
+             ] = PlacesCache.all(nil, return: :value)
+    end
+
+    test "adds PA/ESS screens", %{state: state} do
+      expect(Screenplay.Stops.Mock, :fetch_parent_stops, 2, fn _stop_ids ->
+        [
+          %{
+            "id" => "70036",
+            "attributes" => %{"name" => "Oak Grove"},
+            "relationships" => %{"parent_station" => %{"data" => %{"id" => "place-ogmnl"}}}
+          },
+          %{
+            "id" => "Oak Grove-01",
+            "attributes" => %{"name" => "Oak Grove"},
+            "relationships" => %{"parent_station" => %{"data" => %{"id" => "place-ogmnl"}}}
+          },
+          %{
+            "id" => "Oak Grove-02",
+            "attributes" => %{"name" => "Oak Grove"},
+            "relationships" => %{"parent_station" => %{"data" => %{"id" => "place-ogmnl"}}}
+          },
+          %{
+            "id" => "74615",
+            "attributes" => %{"name" => "World Trade Center - Silver Line - South Station"},
+            "relationships" => %{"parent_station" => %{"data" => %{"id" => "place-wtcst"}}}
+          }
+        ]
+      end)
+
+      expect(Screenplay.Stops.Mock, :fetch_all_parent_stations, 1, fn ->
+        [
+          %{"id" => "place-ogmnl", "attributes" => %{"name" => "Oak Grove"}},
+          %{"id" => "place-wtcst", "attributes" => %{"name" => "World Trade Center"}}
+        ]
+      end)
+
+      expect(Screenplay.Routes.Mock, :fetch_routes_for_stop, 2, fn
+        id when id in ["place-wtcst", "74615", "74613"] ->
+          [
+            %{"id" => "741", "attributes" => %{"type" => 3}},
+            %{"id" => "742", "attributes" => %{"type" => 3}},
+            %{"id" => "743", "attributes" => %{"type" => 3}},
+            %{"id" => "746", "attributes" => %{"type" => 3}}
+          ]
+
+        _ ->
+          [%{"id" => "Orange", "attributes" => %{"type" => 1}}]
+      end)
+
+      assert {:noreply, _} = Builder.handle_info(:build, state)
+
+      assert [
+               %Place{
+                 description: nil,
+                 id: "place-ogmnl",
+                 name: "Oak Grove",
+                 routes: ["Orange"],
+                 screens: [
+                   %Screenplay.Places.Place.PaEssScreen{
```

Comment:
`Place` should be aliased already in this context so we don't have to use the full path. (ditto for L266, L271)